.github/ISSUE_TEMPLATE/questions.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Questions
+about: Ask questions.
+title: ''
+labels: 'type: question, status: waiting-for-feedback, in: UNKNOWN-frontend'
+assignees: ''
+
+---
+
+### Please select a label for the relevant feature.
+
+*Of the labels displayed on the right, change the label that starts with 'in:' to an appropriate one.
+If you don't know, you can leave it as it is.
+(The label may be changed by the maintainer for management purposes. Please note.)*
+
+## System information
+
+ * **Airsonic version**: *the version listed on the About page*
+ * **Operating system**: *the operating system you're running Airsonic on.
```

Comment:
I'm thinking it could be a good idea to also include a line on how Airsonic is run, like "Tomcat/standalone war"?

Reply:
The original format inherits [ISSUE_TEMPLATE.md](https://github.com/airsonic/airsonic/blob/master/.github/ISSUE_TEMPLATE.md). If you wish, we will add to all formats.